packages/eslint-config-4catalyzer/rules.js

```diff
@@ -85,20 +85,27 @@ module.exports = {
       tsx: 'never',
     },
   ],
-  'import/prefer-default-export': 'off',
   'import/order': [
     'error',
     {
-      'newlines-between': 'always',
       groups: [
         'builtin',
         ['unknown', 'external'],
         'internal',
         ['parent', 'sibling', 'index'],
       ],
+      pathGroups: [
+        {
+          pattern: './*.module.{css,scss,less}',
+          group: 'sibling',
+          position: 'after',
+        },
+      ],
+      'newlines-between': 'always',
```

Comment:
what's up with these moving around in an object? just coincidental/random/clean up?

Reply:
alphabetizing rules, sorting options by documented order